assets/src/styles/network.scss

```diff
@@ -0,0 +1,182 @@
+@import 'common';
+
+$fa-font-path: './fonts' !default;
+@import '../../node_modules/@fortawesome/fontawesome-free/scss/fontawesome';
+@import '../../node_modules/@fortawesome/fontawesome-free/scss/brands';
+@import '../../node_modules/@fortawesome/fontawesome-free/scss/solid';
+@import '../../node_modules/leaflet/dist/leaflet';
+@import '../../node_modules/leaflet.markercluster/dist/MarkerCluster';
+@import '../../node_modules/leaflet.markercluster/dist/MarkerCluster.Default';
+
+@import 'partials/script';
+@import 'partials/slider';
+@import 'partials/social-share';
+@import 'partials/tooltip';
+
+.wdfn-component {
+  &[data-component='network'] {
+    @import './components/network';
+    margin-bottom: 1em;
+  }
+}
+
+/* This fixes an issue with Firefox not hiding long divs. I opened https://github.com/uswds/uswds/issues/2380 .
+ * If it gets addressed this can be removed.
+ */
+.usa-sr-only {
+  overflow: hidden;
+}
+
+body {
+  #main-content {
+    margin-top: 1em;
+    margin-bottom: 1em;
+    padding: 0 1em;
+  }
+
+  .content-container {
+    margin-left: auto;
+    margin-right: auto;
+    @include u-maxw('desktop');
+  }
+
+  #wdfn-alert-banner input:checked ~ * {
+    display: none;
+  }
+
+  #wdfn-alert-banner {
+    background-color: color('accent-cool-lighter');
+
+    .wdfn-alert-body {
+      padding-top: 0.5em;
+      padding-bottom: 0.5em;
+      padding-right: 1em;
+      padding-left: 1em;
+      margin-top: 0;
+
+      .close {
+        float: right;
+        cursor: pointer;
+        margin-top: -5px;
+        margin-right: -10px;
+      }
+
+      .div {
+        font-size: 0.9em;
+        @include at-media('tablet') {
+          font-size: 1em;
+        }
+        padding-left: 1em;
+      }
+    }
+
+  }
+}
+
+sort {
```

Comment:
What is this style for? If we are not using it, it should be removed.

Reply:
Table tweaks moved to be specific to network component div.